README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
+
+[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
+> :stars: Navigate the cosmos of single-cell morphology with confidence — coSMicQC keeps your data on course!
+
+coSMicQC is a software tool to evaluate converted single-cell morphology outputs from CytoTable.
+
+When performing object detection in CellProfiler, it is very hard to get "perfect" compartment segmentation across a large high-throughput screen.
```

Comment:
Consider moving to active tense here, avoid possible opinionated language, and pluralize screens (along with object detection software?).
```suggestion
It can be challenging to get "perfect" single-cell compartment segmentation across large high-throughput screens when performing object detection in CellProfiler (or similar software).
```

Reply:
Fair, I think my excitement might have biased the sentence there, thanks for catching! I will accept the change!